web/app/js/components/util/ApiHelpers.jsx

```diff
@@ -1,12 +1,34 @@
+import _ from 'lodash';
+import { Link } from 'react-router-dom';
+import React from 'react';
 import 'whatwg-fetch';
 
-export const ApiHelpers = pathPrefix => {
-  const podsPath = `${pathPrefix}/api/pods`;
+export const ApiHelpers = (pathPrefix, defaultMetricsWindow = '10m') => {
+  let metricsWindow = defaultMetricsWindow;
+  const podsPath = `/api/pods`;
+  const metricsPath = () => `/api/metrics?window=${metricsWindow}`;
+
+  const validMetricsWindows = {
+    "10s": true,
+    "1m": true,
+    "10m": true,
+    "1h": true
+  };
```

Comment:
agree with your comment that `10s` may not make sense, particularly if our collection interval is never less than that.

assuming smallest collection interval is `10s`, a window of `1m` still only gives 6 data points. assuming our graphs can always handle >=30 data points, perhaps our smallest window should be `5m`?

also assuming our telemetry eventually scales, i think a max window of `24h` or greater could be useful.

Reply:
Yeah, that seems reasonable. From `api.proto` we currently have:
```
enum TimeWindow {
  TEN_SEC = 0;
  ONE_MIN = 1;
  TEN_MIN = 2;
  ONE_HOUR = 3;
}
```
Happy to update this pending telemetry improvements. Will make a ticket and revisit in the future.